### ISSUE TITLE
Need "centers" when describing distance in dynamic state of the sim list

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#112]
@mbarlow12 thanks for pointing this out. For scientific accuracy and clarity we should be clear to learners that the distance is measure from the centers of the mass spheres:

Please update the string describing distance from:
- ‪m1‬ and ‪m2‬ are {{somewhat close to each other}}, exactly {{4}} ‪meters‬ apart.
To: 
- m1 and m2 are {{somewhat close to}} each other, centers exactly {{4}} meters apart.

**Note**
- We will need a similar wording change in GFLB. 
- Also note that distance is never turned off in the PDOM in GFL as it is in GFLB.

**Comment by terracoda** [comment]
Sounds good @mbarlow12. I verified on master. Thanks, closing.
